stl/inc/yvals_core.h

```diff
@@ -415,6 +411,23 @@
 #include <vcruntime.h>
 #include <xkeycheck.h> // The _HAS_CXX tags must be defined before including this.
 
+#define _STL_PRAGMA(PRAGMA)          _Pragma(#PRAGMA)

```

Comment:
The CUDA and Intel compilers had trouble with Standard `_Pragma`:

https://github.com/microsoft/STL/blob/04ee87830a404437f1691d6a3e8809e4986d8493/stl/inc/yvals_core.h#L590-L593

I don't know if this changed for CUDA 11.6 or if it's still a problem for Intel. Until we know for sure, I'd recommend making this switch between `_Pragma` and `__pragma`.